index.html

```diff
@@ -5,7 +5,6 @@
     <title>Ofek Twitter</title>
     <link rel="stylesheet" href="css/normalize.css" />
     <link rel="stylesheet" href="css/header.css" />
-    <link rel="stylesheet" href="css/footer.css" />
     <link rel="stylesheet" href="css/index.css" />
 </head>
 <body>
```

Comment:
line 41 - `<strong>` used to indicate important parts of a sentence, not for style - use CSS for styling